README.md

```diff
@@ -27,6 +27,12 @@ To use the dev profile, which contains all dev/repl/test stuff, use the `:dev` a
 * `test-conformance` - Run all conformance tests.
 * `test-all` - Run all lib and conformance tests.
 
+## Implementation Notes
+
+### Async `GET /statements` Error Handling
```

Comment:
Excellent. These notes should also include other "quirks" in the lrs lib, not just this one, but that's outside the scope of this PR.